corgi/api/views.py

```diff
@@ -646,12 +645,17 @@ def manifest(self, request: Request, uuid: str = "") -> Response:
         manifest = json.loads(obj.manifest)
         return Response(manifest)
 
-    @action(methods=["put"], detail=True)
-    def olcs_test(self, request: Request, uuid: Union[str, None] = None) -> Response:
+    @action(
+        methods=["put"],
+        detail=True,
+        authentication_classes=[TokenAuthentication],
+        permission_classes=[IsAuthenticatedOrReadOnly],
+    )
+    def update_license(self, request: Request, uuid: Union[str, None] = None) -> Response:
         """Allow OpenLCS to upload copyright text / license scan results for a component"""
         # In the future these could be separate endpoints
         # For testing we'll just keep it under one endpoint
-        if utils.running_prod() or settings.COMMUNITY_MODE_ENABLED:
+        if settings.COMMUNITY_MODE_ENABLED:
             # This is only temporary for OpenLCS testing
             # Do not enable in production until we add OIDC authentication
             return Response(status=status.HTTP_403_FORBIDDEN)
```

Comment:
We can remove this check and failure for COMMUNITY_MODE_ENABLED too as the endpoint I think. Since we don't have CORGI_OIDC_AUTH_ENABLED set in community environment the OIDC_AUTH_ENABLED django setting is correctly being set to False. That should mean we always get a 403 for this endpoint right?

```
$ oc project
Using project "corgi--community" on server "https://api.mpp-e1-preprod.syvu.p1.openshiftapps.com:6443".
$ oc rsh dc/corgi-web
$ python3 manage.py shell
>>> from django.conf import settings
>>> settings.OIDC_AUTH_ENABLED
0
```

Reply:
Nope, @juspence is correct in that token authentication would still nominally let users of the community instance write if they authenticated since OIDCAuthentication is not currently enabled, however, since we don't have any users in our community instances, it shouldn't be an issue.